scripts/content-types/load-content-types.ts

```diff
@@ -0,0 +1,56 @@
+import { uploadSchema } from './lib/api';
+import { parseSchemas } from './lib/parse-schemas';
+import { apiKeyPrompt, confirmPrompt, getRequiredParam } from './lib/utils';
+
+// These options should not typically be used, so set as env vars instead of regular args
+const JWP_HOST = process.env.JWP_HOST || 'https://api.jwplayer.com';
+const AUTO_CONFIRM = process.env.AUTO_CONFIRM === '1';
+const AUTO_REPLACE = process.env.AUTO_REPLACE === '1';
+
+async function run() {
+  console.info('-------------------------------------------------------');
+  console.info('|           Load Content Types                        |');
+  console.info('-------------------------------------------------------');
+
+  const args: { [key: string]: string } = Object.fromEntries(
+    process.argv.filter((arg) => arg.startsWith('--') && arg.includes('=')).map((arg) => arg.substring(2).split('=', 2)),
+  );
+
+  if (args['help'] || Object.keys(args).length === 0) {
+    console.info('\r\nThis script can be used to upload content types to your JWP environment');
+    console.info('\r\n\r\nUsage: ts-node load-content-types --site-id=<JWP site ID> --api-key=<JWP V2 API Key> [optional arguments]');
+    console.info('\r\n\r\nRequired argument: site-id - specify the JWP property ID to upload to');
+    console.info('Optional argument: source-file - specify a schema file to upload (defaults to ./content-types.json');
+    console.info(
+      'Optional argument: filter - specify a CSV list of schema names in order to limit the schemas to be uploaded to a subset of those found in the source file',
+    );
+
+    process.exit(0);
+  }
+
+  const file = args['source-file'] || `${__dirname}/content-types.json`;
+  const apiKey = await apiKeyPrompt();
```

Comment:
Probably it makes sense to make it required param.

Reply:
I intentionally removed it because it was a security leak to keep pasting it into the terminal. It's annoying, but more secure to prompt for it each time.